aiida_vasp/calcs/tests/test_immigrant.py

```diff
@@ -61,3 +61,58 @@ def test_vasp_immigrant(immigrant_with_builder):
 
     expected_output_nodes = {'misc', 'remote_folder', 'retrieved'}
     assert expected_output_nodes.issubset(set(result))
+
+
+@pytest.fixture
+def immigrant_with_builder_obsv3(fresh_aiida_env, potcar_family, phonondb_run, localhost, mock_vasp):
```

Comment:
Can we replace `obsv3` with something else? Like `example_3` or something like that? I know pylint complains now (in fact, we might consider a global change of that). I do not mind longer names with todays monitors and resolution.

Reply:
Thansk. Yes, it was longer name and I changed due to pylint. I will change it to a longer name with omitting pylint complains.